app/assets/javascripts/components/HomeLogin.jsx

```diff
@@ -0,0 +1,116 @@
+class HomeLogin extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      error: '',
+      hasError: false
+    }
+  };
+
+  _getToken() {
+    const token = document.getElementsByName("csrf-token")[0].getAttribute("content");
+    return token;
+  };
+
+  _handleKeydown(k) {
+    if (k.keyCode == 13) {
+      this._handleLogin();
+    }
+  }
+
+  handleChange(event) {
+    this.setState({ [$(event.target).attr("name")] : $(event.target).val() });
```

Comment:
Instead of using jquery to get the form name, we can just manually setstate for email and password.

Reply:
Also we can use a ref or a controlled form component to get the value of the text inputs instead of jquery